assets/check

```diff
@@ -70,5 +75,5 @@ fi
 echo 'Release not found'
 echo 'Fetching the latest release instead'
 
-curl --silent --show-error --fail --header "Authorization: apiToken ${api_token}" "${endpoint}/api/releases" \
+curl --silent --show-error --fail --user-agent "instana/pipeline-feedback-resource/${version}" --header "Authorization: apiToken ${api_token}" "${endpoint}/api/releases" \
```

Comment:
**[question]** All other places have the `v` prefix for versions. Why not here?

Reply:
I removed the v in too few places

Reply:
Addressed in #242e75cacd00ef61e3e2acc4fa08825337dc4ad3